library/libraries.js

```diff
@@ -660,7 +660,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
 
     'ControlJS': {
         icon: 'icon_48',
-        url: 'http://stevesouders.com/controljs/',
+        url: 'http://controlsjs.com/',
```

Comment:
This is the wrong URL for this library.  The original is still online and refers to the correct library detection test.

If you'd like, you can add an entry to include the library used at [controlsjs.com](http://controlsjs.com) and send an additional PR.